stactools_planet/stactools/planet/planet_item.py

```diff
@@ -91,13 +132,45 @@ def to_stac(self):
             if asset_type != bundle_type:
                 key = '{}:{}'.format(bundle_type, asset_type)
 
-            item.add_asset(key, pystac.Asset(href=href, media_type=media_type))
+            item.add_asset(key, pystac.Asset(href=href, media_type=media_type, roles=roles))
+            asset = pystac.Asset(href=href, media_type=media_type)
+
+            if media_type == pystac.MediaType.COG:
+                #add bands to asset
+                if item_type.startswith('SkySat'):
+                    if "panchro" in asset_type:
+                        bands = [SKYSAT_BANDS['PAN']]
+                    elif "analytic" in asset_type:
+                        bands = [
+                                 SKYSAT_BANDS['BLUE'],
+                                 SKYSAT_BANDS['GREEN'],
+                                 SKYSAT_BANDS['RED'],
+                                 SKYSAT_BANDS['NIR']
+                                ]
+                    else:
+                        bands = [
+                                 SKYSAT_BANDS['RED'],
+                                 SKYSAT_BANDS['GREEN'],
+                                 SKYSAT_BANDS['BLUE']
+                                ]
+                    item.ext.eo.set_bands(bands, asset)
+
+            item.add_asset(key, asset)
+
+        # proj
+        if 'epsg_code' in props:
+            item.ext.enable('projection')
+            item.ext.projection.epsg = props.pop('epsg_code')
+            if geotransform is not None:
+                item.ext.projection.transform = geotransform
+                item.ext.projection.shape = [width, height]
```

Comment:
Shape should be `[height, width]`

```suggestion
                item.ext.projection.shape = [height, width]
```